pkg/machines/components/create-vm-dialog/createVmDialog.jsx

```diff
@@ -195,6 +242,19 @@ class CreateVM extends React.Component {
                         </Select.SelectEntry>))
         );
 
+        let volumeEntries;
+        if (this.state.storagePool) {
+            const pool = this.props.storagePools.find(pool => pool.name === this.state.storagePool && pool.connectionName === this.state.connectionName);
+            if (pool && pool.volumes) {
+                volumeEntries = (
+                    pool.volumes.map(vol => {
+                        if (vol.type === "file" || vol.type === "network" || vol.type === "block")
```

Comment:
Yes after discussion with libvirt guys, they agreed other types will cause error when passed to virt-install, or are "troublesome to implement here".

Reply:
But basically this means that virt-install does not support other pool types... Is this the case? In any case this will need to be commented with pointers to the documentation.
I mean man virt-install, search for vol will not state any limitation...

Reply:
Funny thing, there is no documentation about which pool types are supported, nor any pointers.
You can only learn from libvirt code that netdir and dir volume types won't work in this case, whilst file, network and block will work. So I really don't know what comment can I leave there, apart from "these types will work".

Reply:
So final decision is to do no filtering at all.